diesel_tests/tests/insert.rs

```diff
@@ -66,28 +66,20 @@ fn insert_with_defaults() {
 }
 
 #[test]
-#[should_panic] // FIXME: SQLite has no DEFAULT keyword. We need to work around this.
-#[cfg(feature = "sqlite")] // FIXME: This test should run on everything, the only difference is create table syntax.
+#[cfg(feature = "postgres")] // FIXME: This test should run on everything, the only difference is create table syntax.
```

Comment:
The diff here rendered in a not terribly helpful way. The change was that I renamed our existing `insert_with_defaults` test to `batch_insert_with_defaults`, and set it to run on everything except SQLite. I then changed the `insert_with_defaults` test on both backends to operate on a single record instead of a batch.